src/view/dataElements/identityMap.jsx

```diff
@@ -407,30 +383,19 @@ function IdentityMap({ initInfo, formikProps, registerImperativeFormApi }) {
           );
         }}
       />
-    </React.Fragment>
+    </>
   );
-}
-
-IdentityMap.propTypes = {
-  initInfo: PropTypes.object,
-  formikProps: PropTypes.object,
-  registerImperativeFormApi: PropTypes.func
 };
```

Comment:
How did you get away with removing prop types?

Reply:
I removed all the props on the component so there is no need to declare propTypes.